cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -768,6 +769,175 @@ func TestEvaluateExecute(t *testing.T) {
 		})
 	})
 
+	t.Run("Runtime", func(t *testing.T) {
+		// Do not execute Docker tests if it is not available
+		if _, err := exec.LookPath("docker"); err != nil {
+			t.Skip("Docker runtime not found")
+		}
+
+		if osutil.IsWindows() {
+			t.Skip("Docker runtime not supported on Windows")
+		}
+
+		validate(t, &testCase{
+			Name: "Docker",
+
+			Arguments: []string{
+				"--runtime", "docker",
+				"--model", "symflower/symbolic-execution",
+				"--model", "symflower/symbolic-execution",
+				"--model", "symflower/symbolic-execution",
+				"--testdata", "testdata/", // We need to override the testdata path back to the default one because the actual evaluation is running inside the container.
```

Comment:
Because it gets set inside the `validation` function `cmd/eval-dev-quality/cmd/evaluate_test.go:140` it is a default which is used everywhere except for the containerized stuff because we do not have any temporary testdata path there but the testdata from the image itself.

Reply:
Maybe then change the docstring to `Our own tests set the "testdata" argument to the temporary directory that they create. This temporary directory does not exist in docker, so set the "testdata" manually here to overrule the testing behavior and use the original one.`